assets/src/scss/editorStyle.scss

```diff
@@ -41,7 +41,7 @@
 @import "blocks/Columns/ColumnsEditorStyle";
 @import "blocks/Cookies/CookiesEditorStyle";
 @import "blocks/ENForm/ENFormEditorStyle";
-@import "blocks/Happypoint/HappypointEditorStyle";
+@import "blocks/HappyPoint/HappypointEditorStyle";
```

Comment:
The style file's case should also be fixed: `HappypointEditorStyle` => `HappyPointEditorStyle`